Clean up h/g/p wording (#5341)
Qiskit/documentation: docs/guides/considerations-set-up-runtime.mdx

# Considerations for setting up IBM Quantum Platform for an organization

> IBM Quantum Platform is the dashboard for your IBM Cloud&reg; account's Qiskit Runtime service instances and workloads, and provides a simplified view of access management. An organization's IBM Cloud account can have multiple users and multiple Qiskit Runtime service instances, each with its own allocation. Identity and Access Management (IAM) controls which users can access which service instances, so you can enable collaboration while restricting visibility where needed. Access management becomes more relevant if you have service instances on paid plans. See [IBM Cloud account structure](/docs/guides/cloud-account-structure) for an overview of how accounts, users, instances, and access fit together. Refer to the [IBM Cloud IAM documentation](https://cloud.ibm.com/docs/account?topic=account-iamoverview) for full details on the IAM concepts referenced in this guide, such as access groups, policies, roles, and resource groups.

Reviewer: ```suggestion
IBM Quantum&reg; Platform is the dashboard for your IBM Cloud&reg; account's IBM Quantum Compute Service instances and workloads, and provides a simplified view of access management. An organization's IBM Cloud account can have multiple users and multiple Quantum Compute instances, each with its own allocation. Identity and Access Management (IAM) controls which users can access which service instances, so you can enable collaboration while restricting visibility where needed. Access management becomes more relevant if you have service instances on paid plans. See [IBM Cloud account structure](/docs/guides/cloud-account-structure) for an overview of how accounts, users, instances, and access fit together. Refer to the [IBM Cloud IAM documentation](https://cloud.ibm.com/docs/account?topic=account-iamoverview) for full details on the IAM concepts referenced in this guide, such as access groups, policies, roles, and resource groups.
```

> This guide describes the decisions and tradeoffs involved in setting up access for an organization with multiple service instances. For example, having one instance per team or workload.

Reviewer: ```suggestion
This guide describes the decisions and tradeoffs involved in setting up access for an organization with multiple service instances &mdash; for example, assigning one instance per team or workload.
```

<Admonition type="note">

> If your organization has multiple IBM Cloud accounts - for example, separate accounts per business unit, each with its own Qiskit Runtime service instances - you can link them under a single IBM Cloud Enterprise account, which has one main account responsible for billing and one or more child accounts. To redistribute Premium or Flex Plan allocation between child accounts, contact IBM Quantum support through the [IBM Cloud Support Center](https://cloud.ibm.com/unifiedsupport/cases/form). See the [IBM Cloud Enterprise account documentation](https://cloud.ibm.com/docs/account?topic=account-what-is-enterprise) for full details.

Reviewer: ```suggestion
If your organization has multiple IBM Cloud accounts &mdash; for example, separate accounts per business unit, each with its own Quantum Compute instances &mdash; you can link them under a single IBM Cloud Enterprise account, which has one main account responsible for billing and one or more child accounts. To redistribute Premium or Flex Plan allocation between child accounts, contact IBM Quantum support through the [IBM Cloud Support Center](https://cloud.ibm.com/unifiedsupport/cases/form). See the [IBM Cloud Enterprise account documentation](https://cloud.ibm.com/docs/account?topic=account-what-is-enterprise) for full details.
```

</Admonition>

<span id="overview-org"></span>
## Overview

   <Admonition type="note">

>    IBM Cloud&reg; provides several ways to implement the mechanisms described in this guide. Most of the steps are generic to IBM Cloud and not specific to Qiskit Runtime, except the custom role details.

Reviewer: ```suggestion
   IBM Cloud&reg; provides several ways to implement the mechanisms described in this guide. Most of the steps are generic to IBM Cloud and not specific to Quantum Compute, except the custom role details.
```

</Admonition>

### Involved personas

> There are several main personas mentioned in this guide:

Reviewer: ```suggestion
The following personas are mentioned in this guide:
```

* **User**: Someone who gets access to Qiskit Runtime resources (_service instances_) and can potentially collaborate with other users on these resources. Users' access is controlled by an administrator, and they cannot create or delete service instances.

> * **Cloud administrator**: An IBM Cloud account owner who owns the Qiskit Runtime resources and manages which users can access these resources. As the resource owner, the administrator is charged for any paid resource use.

Reviewer: ```suggestion
* **User**: Someone who gets access to Quantum Compute resources (_service instances_) and can potentially collaborate with other users on these resources. Users' access is controlled by an administrator, and they cannot create or delete service instances.
* **Cloud administrator**: An IBM Cloud account owner who owns the Quantum Compute resources and manages which users can access these resources. As the resource owner, the administrator is charged for any paid resource use.
```

* **IDP administrator**: An administrator who defines identities and their attributes in an identity provider (IDP).

### Terminology

This guide uses the following terms:

> * _Resource_: A generic IBM Cloud term that refers to an object that can be managed through the Cloud user interface, CLI, or API. For this guide, a _resource_ is a Qiskit Runtime service instance.
> * _Service instance_: A service instance is used to access Cloud services - specifically, quantum computers, through the Qiskit Runtime service. It is defined through the catalog. You can define several service instances based on the same or different plans, which offer access to different quantum computing backends. See [Available IBM Cloud plans](/docs/guides/plans-overview) for details.

Reviewer: ```suggestion
* _Resource_: A generic IBM Cloud term that refers to an object that can be managed through the Cloud user interface, CLI, or API. For this guide, a _resource_ is a Quantum Compute instance.
* _Service instance_: A service instance is used to access Cloud services - specifically, quantum computers, through the IBM Quantum Compute Service. It is defined through the catalog. You can define several service instances based on the same or different plans, which offer access to different quantum computing backends. See [Available IBM Cloud plans](/docs/guides/plans-overview) for details.
```

<span id="xplanning-orgx"></span>
## Plan your setup

> Before you set up IBM Quantum Platform for your organization, you need to make these decisions:

Reviewer: Is this use of Platform ok, or should it be changed to Quantum Compute? (I ask because of the changes from Platform to Qiskit Runtime I saw above)

Author: I think I'll leave it since it matches the title.

# Considerations for setting up IBM Quantum Platform for an organization

> In an organization where individuals might work on several projects, IBM Quantum Platform governance can seem complex. However, access management can be used to easily enable user collaboration and to restrict visibility of users and projects when necessary. Managing access becomes more relevant with IBM Quantum Platform resources that are not free - that is, service instances that use paid plans (which organizations are charged for).  In this guide it is assumed that you set up a separate instance for each internal project.

Reviewer 2: ```suggestion
In an organization where individuals might work on several projects, IBM Quantum Platform governance can seem complex. However, access management can be used to easily enable user collaboration and to restrict visibility of users and projects when necessary. Managing access becomes more relevant with IBM Quantum Platform resources that are not free - that is, service instances that use paid plans (which organizations are charged for).  In this guide, it is assumed that you set up a separate instance for each project.
```

Reviewer 2: Idk if we should hedge with something like ", which is only one possible way to set up instances."

When using dynamic rules, that is, when the identity provider (IDP) administrator manages access through custom IDP user attributes, do not use IDP custom user attributes that are substrings of each other. For instance, don't use `ml` and `mlReader`, as the string comparison of `ml` would also accept `mlReader`. You could use `MLreader` and `MLwriter` to avoid this conflict.
</Admonition>

For an example, see [Set up custom roles](/docs/guides/custom-roles).

<span id="shared-access"></span>
### Shared workload access

It is important to note that access applies to service instances.  Thus, users with 'write' access to an instance can cancel their own workloads, but can also view and cancel other users' workloads.  This is a function of how IAM works and cannot be changed.

<span id="nest-org"></span>
### Hierarchical project structures

> In this guide, the mapping of users to projects and service instances was kept simple. However, by associating several users with access groups and referencing service instances from several access groups, more complex mappings can be implemented.

Reviewer 2: Per https://github.com/Qiskit/documentation/issues/5158, I continue to find it confusing to talk about "Projects". I think it would be better for us to primarily use "instances", other than maybe having an early definition explaining that you can roughly think of instances like projects.

Author: I got rid of instances of "project" in this topic, but we can revisit these topics in greater detail in #5158.
